src/test/java/com/microsoft/sqlserver/jdbc/connection/ConnectionDriverTest.java

```diff
@@ -332,7 +332,7 @@ public void testDeadConnection() throws SQLException {
                 assertEquals(e.getMessage(), TestResource.getResource("R_connectionReset"),
                         TestResource.getResource("R_unknownException"));
             }
-            assertEquals(conn.isValid(5), false, TestResource.getResource("R_deadConnection"));
+            assertEquals(conn.isValid(0), false, TestResource.getResource("R_deadConnection"));
```

Comment:
Do we want to test isValid with infinite timeout? The method states that if the connection is dead, this method can hang forever.